### ISSUE TITLE
Gesture-Typing Improvement: Turn on Automatic Capitalization

### ISSUE THREAD

**Issue by Glitchy-Tozier** [AnySoftKeyboard/AnySoftKeyboard#1953]
### Problem
Currently, automatic capitalization is not automatically turned on when Gesture-Typing. Due to this, it's straight up impossible to write properly formatted sentences if solely relying on Gesture-Typing as you **need** to use regular typing to write the first word of each sentence and proper nouns.

### Expected behaviour / Possible solutions
There are a couple of them:

### Actual behaviour
Tell us what happens instead

**Android OS version:**

**Device manufacturer and model:**

**List of installed add-ons (like languages, or themes):**

**Comment by TimMagee** [comment]
I think number 3 should be the default as it is what I think people would expect. Every single time (even though I know that it doesn't work) I automatically try to hit shift and expect the first letter of the next word to be capitalized. Not only is this how I would expect it to work, it is how I intuitively attempt to use the interface even though I know it doesn't work.

**Comment by Glitchy-Tozier** [comment]
Exactly!
Also, I would at least add the option of enabling a separate autocomplete.
This gives users the option of having a setup similar to the default Gesture-Typing-behaviour many regular keyboards display. (Samsung and, I think, Google also auto-capitalizes when Swiping)

So I'd suggest number 2 + number 3 with the automatic capitalization option being turned on by default.

**Comment by Glitchy-Tozier** [comment]
I realized, this is an important improvement.
Languages such as German are very reliant on capitalization as in German **every noun** is capitalized. Which makes the current Gesture-Typing almost unbearable.

**Comment by eatmeek** [comment]
This is a result important issue.  I am moving to AnySoftKeyboard from Gboard as it offers swipe / gesture topping and overall it's a pretty  good alternative but using shift/caps to capitalise while gesture typing  is essential.

Number  3 is the most important solution but  I would support all three.

**Comment by menny** [comment]
Something like that?
https://user-images.githubusercontent.com/481272/197866365-a4b0b99f-e974-4af3-9c29-5521b8060d16.mp4

**Comment by dbolton** [comment]
@menny Yes! Is your video of a development version of AnySoftKey or another keyboard?

**Comment by menny** [comment]
Yes, this is the latest AnySoftKeyboard from development. The next release will have all of this.
You can get this now if you register to the BETA channel
